.github/workflows/Documenter.yml

```diff
@@ -5,17 +5,22 @@ on:
       - master
     tags: '*'
   pull_request:
-
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@latest
         with:
           version: '1'
+      - name: Install dependencies
+        shell: julia --color=yes --project=docs {0}
+        run: |
+          using Pkg
+          Pkg.develop(PackageSpec(path=pwd()))
+          Pkg.instantiate()
       - name: Build and deploy
+        run: julia --color=yes --project=docs docs/make.jl
```

Comment:
shouldn't we be using `julia-actions/julia-docdeploy@latest`?

Reply:
actually that's probably even better!

Reply:
done; [e1a7d7e](https://github.com/KristofferC/OhMyREPL.jl/pull/379/commits/e1a7d7e80bb6e41c4f0fe3a624948f2175c19fe8)